src/kata6.joinNames.js

```diff
@@ -1,3 +1,9 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  const names = namesObj.map((anything) => anything.nom);
```

Comment:
I know I mentioned in a previous comment you could use anything as the placeholder, I meant that as anything you choose and not literally the word 'anything' :D 
It should be something that accurately represents what the value is, so in this case name would be fine:
```
const names = namesObj.map((person) => person.nom);
```